kusto/internal/frames/v1/v1.go

```diff
@@ -39,7 +39,11 @@ func (dt DataType) toColumn() (table.Column, error) {
 	col := table.Column{Name: dt.ColumnName}
 
 	if dt.ColumnType != "" {
-		col.Type = types.Column(dt.ColumnType)
+		var ok bool
```

Comment:
I think I want to make a cleanup to this and cover an extra error condition we aren't covering.  

You can find my version at:
https://go.dev/play/p/eEez1fBQ9M-

I also am including a test there that can go in the test file.  I haven't run this, but if it is off it should be barely off.

Again, thanks for doing this Jesse.

Reply:
Figured I'd ping on this issue, see if you still want to make the changes and push over.

Reply:
hey @element-of-surprise, I have now pushed the updated commit (with changes + unit test) into this branch